docs/cloud-tutorials/byod-management.md

```diff
@@ -0,0 +1,101 @@
+---
+sidebar_position: 5
+title: Bringing Your Own Database
+description: Learn how to bring your own PostgreSQL database to DBOS Cloud
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+In this guide, you'll learn how to bring your own PostgreSQL database instance to DBOS Cloud and deploy your applications to it.
+
+:::info
+This feature is currently only available to [DBOS Pro/Custom plan](https://www.dbos.dev/pricing) subscribers.
```

Comment:
I suggest converging on the names for the plans and using them without "/" - so the users know exactly what to buy in order to get the feature. If "Pro" and "Custom" are separate plans then I would spell it out as "Pro or Custom"